src/testing/code-debugging.md

```diff
@@ -274,8 +283,8 @@ widget's description.
 ### Render tree
 
 If you are trying to debug a layout issue, then the Widgets layer's
-tree might be insufficiently detailed. In that case, you can dump the
-rendering tree by calling [`debugDumpRenderTree()`][].
+tree might be insufficiently detailed. To dump the rendering tree,
+call the [`debugDumpRenderTree()`][] method.
```

Comment:
Since it's not a class or instance function, it is not a method, just a function.

```suggestion
call the [`debugDumpRenderTree()`][] function.
```